strangelove-ventures/wadesta
# feat(cli): generate ChainInfo output

## Description

Generate cli output as keplr's `ChainInfo`

## Checklist


## Changes

- [x] [feat(cli): generate ChainInfo output](https://github.com/strangelove-ventures/graz/commit/5c84f99a8e8287cd00770b0f8e03b862740e2ad7)

## Screenshots
![CleanShot 2022-09-02 at 01 21 50](https://user-images.githubusercontent.com/39829726/187985589-26e63cb8-227c-47ca-a193-7323536c729b.png)


## Testing

- run `pnpm example graz -g`
- see the output
